src/components/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -45,6 +45,19 @@ const PipelineListRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj }) =>
       <TableData className={pipelineRunTableColumnClasses.type}>
         {capitalize(obj.metadata?.labels[PipelineRunLabel.PIPELINE_TYPE])}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.component}>
+        {obj.metadata?.labels[PipelineRunLabel.APPLICATION] ? (
+          <Link
+            to={`/stonesoup/applications/${
+              obj.metadata?.labels[PipelineRunLabel.APPLICATION]
+            }/components`}
+          >
+            {capitalize(obj.metadata?.labels[PipelineRunLabel.COMPONENT])}
+          </Link>
+        ) : (
+          capitalize(obj.metadata?.labels[PipelineRunLabel.COMPONENT])
+        )}
+      </TableData>
```

Comment:
Add another condition in case the component label is not present. We should show '-' in case the component is missing.